package.json

```diff
@@ -10,9 +10,9 @@
     "start": "grunt start",
     "build": "grunt build",
     "eslint": "eslint .",
-    "test": "jest tests/ --runInBand --coverage",
-    "test-no-coverage": "jest tests/ --runInBand",
-    "test-ci": "npm run test && cat ./coverage/lcov.info | coveralls",
+    "test": "jest tests/ --runInBand --watch",
```

Comment:
I suppose one of the other goals I was aiming for was to try and make the test command setup more consistent between addons-frontend and addons-linter since that's another difference we can eliminate. 

If I added the `test-once` command would that provide what you need? See https://github.com/mozilla/addons-frontend/#development-commands for more details.